config/default.json

```diff
@@ -109,7 +109,7 @@
               "queue.buffering.max.messages": 10000000
             },
             "topicConf": {
-              "request.required.acks": "all",
+              "request.required.acks": "1",
```

Comment:
I would be interested in understanding the reason for the switch here.

Reply:
That was likely a mistake, and we can probably remove it.